create-ladder.html

```diff
@@ -2,89 +2,95 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width">
     <title>Exposure Ladder</title>
-    <link rel="stylesheet" href="css/main.css">
-    <!-- <link rel="stylesheet" href="css/create-ladder.css">
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script> -->
-  </head>
+    <link rel="stylesheet" href="css/create-ladder.css">
+      <link href="https://fonts.googleapis.com/css?family=Kranky" rel="stylesheet">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
   <body>
-      <form id="ladder" class="ladder">
-        <button class="add">+</button>
 
-        <form class="challenge" id="challenge-1">
-            <div class="challenge__contents" id="description-1">
-              <label>Description</label>
-              <input/>
-            </div>
-            <div class="challenge__contents" id="anxiety-rating-1">
-              <label>How anxious are you feeling?</label>
-              <input type="range"  min="0" max="100" value="50"/>
-            </div>
-            <button type="submit" class="save">Save</button>
-        </form>
+    <img id='cloud' src='images/cloud.svg'/>
+    <h2 class='cloud-title'>Eating lunch in the Canteen</h2>
 
-        <form class="challenge" id="challenge-2">
-            <div class="challenge__contents" id="description-1">
-              <label>Description</label>
-              <input/>
-            </div>
-            <div class="challenge__contents" id="anxiety-rating-1">
-              <label>How anxious are you feeling?</label>
-              <input type="range"  min="0" max="100" value="50"/>
-            </div>
-            <button type="submit" class="save">Save</button>
-        </form>
+      <div id="ladder" class="ladder">
+          <form class="challenge fold" id="challenge-1">
+              <div class="challenge__contents" id="description-1">
+                <label>Description</label>
+                <input/>
+              </div>
+              <div class="challenge__contents" id="anxiety-rating-1">
```

Comment:
Lots if `ids` here, be careful, they classes are normally better.

On the whole we if we have ids like `thing-1`, `thing-2`, `thing-3` we are over specifiying. $(''.challenge")[3] or something will probable suffice as a selector.